From 9cde0a1303ea805421179b8c30c4837b0bb38a35 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jordan@centricular.com>
Date: Thu, 7 Jul 2022 16:09:21 +0300
Subject: [PATCH] components/polkit: Use the proper meson-local variable for
 kind:meson

This is a leftover from when the element was switched to meson,
and as such the extra arguments we wanted to pass were ignored.

Close #1445
---
 elements/components/polkit.bst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/elements/components/polkit.bst b/elements/components/polkit.bst
index ff576efd57..79ee6490f1 100644
--- a/elements/components/polkit.bst
+++ b/elements/components/polkit.bst
@@ -13,8 +13,8 @@ depends:
 - components/systemd.bst
 
 variables:
-  conf-local: >-
-    -Dsession-tracking=libsystemd-login
+  meson-local: >-
+    -Dsession_tracking=libsystemd-login
     -Djs_engine=duktape
   local_flags: -std=gnu++17
 
-- 
GitLab

